### PR TITLE
[TRANSLATION] update indonesian translation

### PR DESCRIPTION
**Pull request by Mas4hmad**

### CODE REVIEW

File: src/lang/id_ID.rc [hunk 1]
```diff
@@ -86,7 +86,7 @@ IDR_MAINMENU MENU
         MENUITEM "Pengaturan &Fon...", ID_FONTS
         MENUITEM "Tentukan &Jalur...", ID_SETPATHS
         MENUITEM SEPARATOR
-        MENUITEM "&UI Language Select (UI语言选择)...", ID_CHOOSEUILANG
+        MENUITEM "Pilih Bahasa &UI", ID_CHOOSEUILANG
```
Comment (katahiromz) [inline src/lang/id_ID.rc:89]: Please don't translate this menu item.

**Review by katahiromz — CHANGES_REQUESTED** [review]

File: src/lang/id_ID.rc (hunk 1 above)
Comment (Mas4hmad) [reply src/lang/id_ID.rc:89]: may i change the english one only? i can keep the japanese one.

Comment (katahiromz) [reply src/lang/id_ID.rc:89]: Revert it to the original. Not japanese but chinese.

Comment (Mas4hmad) [reply src/lang/id_ID.rc]: thanks

**Review by katahiromz — APPROVED** [review]

**Pull request merged**